testdata/go/projectmissingdependency/projectmissingdependecy.go

```diff
@@ -12,4 +12,4 @@ func execDependency() {
 func Exec() {
 	fmt.Println("Executing ")
 	execDependency()
-}
\ No newline at end of file
+}
```

Comment:
Unnecessary change which adds a redundant new-line.

Reply:
But gofmt makes the go file end with a newline, are you sure not to align this on current file?